pkg/mount/mount.go

```diff
@@ -16,6 +16,7 @@ limitations under the License.
 
```

Comment:
`pkg/mount` is C&P directly from Kubernetes core and we don't want to modify its contents to ensure that syncing with upstream is as simple as possible. The package should be ignored when linting and these changes reverted.

Reply:
Done

Reply:
However, why isn't this code imported, rather than C&P?

Reply:
We don't want to vendor the entirety of Kubernetes for such a small amount of code.